waspc/cli/src/Wasp/Cli/Command/Utils/User.hs

```diff
@@ -0,0 +1,16 @@
+module Wasp.Cli.Command.Utils.User
+  ( getHomeDir,
+    HomeDir,
+  )
+where
+
+import Data.Maybe (fromJust)
+import qualified StrongPath as SP
+import System.Directory
+  ( getHomeDirectory,
+  )
+
+data HomeDir
+
+getHomeDir :: IO (SP.Path' SP.Abs (SP.Dir HomeDir))
```

Comment:
I'd bundle this thing together with `Wasp.Cli.Command.Utils.File` under something like `Wasp.Cli.Command.FileSystem`, dropping the utils folder entirely.

Also, remember StrongPath conventions i mentioned the other day:  https://github.com/wasp-lang/wasp/pull/953#discussion_r1071593511 (drop the `SP.` :).